### PR TITLE
Add candidate view

### PR DESCRIPTION
**Pull request by boonrs**
- Add routes, controller,views. :ski: 
- Make dev pretty :skull:

### CODE REVIEW

File: app/models/candidate.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Candidate < ActiveRecord::Base
+  has_many :answers
```
Comment (thorncp) [inline app/models/candidate.rb]: What do you think about adding `dependent: :destroy` here, so if a candidate is deleted their answers go along with them?

File: app/views/candidates/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<div class="vertical-tabs-container">
+  <div class="vertical-tabs">
+    <% @candidates.each_with_index do |candidate, index| %>
+      <a href="javascript:void(0)" class="js-vertical-tab vertical-tab <%= 'is-active' if index==0 %>" rel="tab<%= candidate.id%>"><%= candidate.name %></a>
```
Comment (thorncp) [inline app/views/candidates/index.html.erb]: Maybe split this to multiple lines so it's easier to see the inner content?

File: app/views/candidates/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <% end %>
+  </div>
+
+  <div class="vertical-tab-content-container">
+    <% @candidates.each do |candidate| %>
+      <a href="" class="js-vertical-tab-accordian-heading vertical-tab-accordion-heading" rel="tab<%= candidate.id %>"><%= candidate.name %></a>
```
Comment (thorncp) [inline app/views/candidates/index.html.erb]: Maybe split this to multiple lines so it's easier to see the inner content?

File: app/views/candidates/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      <div id="tab<%= candidate.id %>" class="js-vertical-tab-content vertical-tab-content">
+        <div class="grid-items">
+          <div class="grid-item">
+            <%= image_tag candidate.photo_url, size: "100x100" %>
+          </div>
+          <div class="grid-item grid-item-big">
+            <% unless candidate.twitter.nil? %>
```
Comment (thorncp) [inline app/views/candidates/index.html.erb]: What do you think about changing this to a positive check? `if candidate.twitter.present?`

File: app/views/candidates/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+              <%= link_to "@#{candidate.twitter}", "http://twitter.com/#{candidate.twitter}" %><br />
+            <% end %>
+
+            <% unless candidate.website.nil? %>
```
Comment (thorncp) [inline app/views/candidates/index.html.erb]: What do you think about changing this to a positive check? `if candidate.website.present?`

File: app/models/candidate.rb (hunk 1 above)
Comment (boonrs) [inline app/models/candidate.rb]: :lollipop: :yum:

File: app/views/candidates/index.html.erb (hunk 2 above)
Comment (boonrs) [inline app/views/candidates/index.html.erb]: :-1: :book:

File: app/views/candidates/index.html.erb (hunk 4 above)
Comment (boonrs) [inline app/views/candidates/index.html.erb]: :point_up: :honey_pot:

**Comment by boonrs** [comment]
closes #12

**Pull request merged**